AUTHORS.rst

```diff
@@ -0,0 +1,80 @@
+Guidelines
+==========
+
+The purpose of this file is to let newcomers know who is who and to whom best address their
+best address questions or ask a review. The format of this file can also be questionned and
+change, do not hesitate to make suggestions!
+
+Ontology experts
+================
+With Ph.D.
+----------
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
+
+With Master
+-----------
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
+
+Researchers
+===========
+
+* @Bachibouzouk
+
+ * 0 year of experience with ontology
+ * Started working on this project since 2019-10-01
+ * Specific relevant area of expertise for this project: github workflow
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
+
+
+Students currently working on the ontology
+==========================================
+During Master
+-------------
+
+* github handle (+ real name if relevant/wanted)
+
+ * X years of experience with ontology
+ * Started working on this project since 20XX-XX-XX
+ * Specific relevant area of expertise for this project: ...
+
+
```

Comment:
```suggestion
* izzet 91 

 * 2 months of experience with ontology
 * Started working on this project since 2019-09-01
 * Specific relevant area of expertise for this project: NA
```